### PR TITLE
Roll web platform tests

### PR DESCRIPTION
**Pull request by domenic**

### CODE REVIEW

File: test/web-platform-tests/to-run.yaml [hunk 1]
```diff
@@ -1271,6 +1281,7 @@ cors-expose-star.sub.any.html: [fail, Unknown]
 data-uri.htm: [fail, Unknown]
 event-error-order.sub.html: [fail, Unknown]
 event-timeout-order.any.html: [fail, Unknown]
+event-upload-progress.any.html: [fail, Unknown]
```
Comment (ExE-Boss) [inline test/web-platform-tests/to-run.yaml:1284]:
```suggestion
event-upload-progress.any.html:
  "Upload events registered on time (http://www1.web-platform.test:9000/xhr/resources/corsenabled.py)": [fail, Unknown]
```

File: test/web-platform-tests/to-run.yaml [hunk 2]
```diff
@@ -1291,6 +1302,7 @@ open-url-worker-origin.htm: [fail, Needs Worker implementation]
 open-url-worker-simple.htm: [timeout, Needs Worker implementation]
 overridemimetype-blob.html: [fail, Unknown]
 overridemimetype-edge-cases.window.html: [fail, Unknown]
+request-content-length.any.html: [fail, needs URL.createObjectURL]
```
Comment (ExE-Boss) [inline test/web-platform-tests/to-run.yaml:1305]:
```suggestion
request-content-length.any.html:
  "Fetched blob: URLs set the Content-Length header": [fail, needs URL.createObjectURL]
  "Uploads need to set the Content-Length header": [fail, needs URL.createObjectURL]
```

**Review by ExE-Boss — COMMENTED** [review]
Now that we have <https://github.com/jsdom/jsdom/pull/3139>, it should be used for test files with failing subtests.

---

**Refs:** <https://github.com/jsdom/jsdom/issues/3157>

**Comment by ExE-Boss** [comment]
Now that we have <https://github.com/jsdom/jsdom/pull/3139>, it should be used for test files with failing subtests.

---

**Refs:** <https://github.com/jsdom/jsdom/issues/3157>

**Comment by domenic** [comment]
@ExE-Boss, you've been asked to please not review pull requests on this repository. This is your second warning.

**Comment by ExE-Boss** [comment]
**Here’s the thing:**

I went out of _my_ way to implement <https://github.com/jsdom/jsdom/pull/3139> with the expectation that it’d be used by the **JSDOM** team for any newly failing test files, but then _you_ open a new PR, after _you_’ve merged <https://github.com/jsdom/jsdom/pull/3139>, that has new test failures that should use the new format:

-	`dom/abort/event.any.html` (1 test failure out of 6 tests)
	```diff
	-event.any.html: [fail, AbortSignal.abort() not implemented]
	+event.any.html:
	+  "the AbortSignal.abort() static returns an already aborted signal": [fail, AbortSignal.abort() not implemented]
	```

-	`xhr/event-upload-progress.any.html` (1 test failure out of 4 tests)
	```diff
	-event-upload-progress.any.html: [fail, Unknown]
	+event-upload-progress.any.html:
	+  "Upload events registered on time (http://www1.web-platform.test:9000/xhr/resources/corsenabled.py)": [fail, Unknown]
	```

But then _you_ decide to just ignore the new format that _I_ implemented in <https://github.com/jsdom/jsdom/pull/3139>, thus increasing the technical debt that <https://github.com/jsdom/jsdom/pull/3139> and <https://github.com/jsdom/jsdom/issues/3157> seeks to alleviate.

---

/cc @TimothyGu
